### PR TITLE
please use my full name

### PR DESCRIPTION
**Pull request by Ruqaiya99**
Please accept this I have made my first contribution

### CODE REVIEW

**Comment by AnimeshN** [comment]
ok

File: readme.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 ## contributors
-* ruqaiya99 - Ruqaiya 
+* ruqaiya99 - Ruqaiya Arjumand
```
Comment (AnimeshN) [inline readme.md:2]: ok

**Review by AnimeshN — CHANGES_REQUESTED** [review]
do you need some more information added

File: readme.md (hunk 1 above)
Comment (Ruqaiya99) [reply readme.md:2]: no thanks

**Comment by Ruqaiya99** [comment]
this is ok

**Pull request merged**